packages/next-auth/src/core/routes/callback.ts

```diff
@@ -190,7 +190,7 @@ export default async function callback(params: {
       }
     } catch (error) {
       if ((error as Error).name === "OAuthCallbackError") {
-        logger.error("CALLBACK_OAUTH_ERROR", error as Error)
+        logger.error("OAUTH_CALLBACK_ERROR", error as Error)
```

Comment:
```suggestion
        logger.error("OAUTH_CALLBACK_ERROR", {
          error: error as Error,
          providerId: provider.id,
        })
```
Let's provide the `providerId` here 🙌

Reply:
Sure. Committing your suggestion 🙂